### ISSUE TITLE
Content2Section is missing its identifier

### ISSUE THREAD

**Issue by tgrassl** [e-Spirit/fsxa-api#18]
Hi, we are currently trying to map a Content2Section, but the identifier is missing in the mapped response. I looked at the raw json response and it looks like the identifier is nested under the template key and does not exist on the initial level.  https://github.com/e-Spirit/fsxa-api/blob/master/src/modules/CaaSMapper.ts#L219-L220

CaaSMapper Response:
```js
{
	name: 'main_content',
	previewId: '378dc8c0-d4eb-4e47-99fc-f9da53fcd097.de_DE',
	children: [
	  {
		id: undefined,
		previewId: '.de_DE',
		type: 'Section',
		data: [Object],
		sectionType: 'public_web.niederlassungen',
		children: []
	  }
	]
}
```

Raw JSON:
```json
...
    {
        "fsType": "Content2Section",
        "schema": "public_web",
        "entityType": "niederlassungen",
        "name": "niederlassung",
        "displayName": "niederlassung",
        "template": {
            "fsType": "TableTemplate",
            "name": "Niederlassungen",
            "displayName": "Niederlassungen",
            "identifier": "ecb5acc9-2c86-4378-98a1-79c19a2a665b",
            "uid": "public_web.niederlassungen",
            "uidType": "TEMPLATESTORE_SCHEMA"
        },
        "query": null,
        "recordCountPerPage": 1,
        "maxPageCount": 0,
        "filterParams": {},
        "ordering": [
            {
                "attribute": "fs_id",
                "ascending": false
            }
        ]
    }
...
```

@tgolbs I opened a PR to fix this issue #17

**Comment by ls-espirit** [comment]
Indeed, the raw JSON of the Content2 Section has currently no identifier. 
I'm think you're proposed change would be semantically incorrect, as the template.identifier does *not* uniquely identify the Content2 Section itself. 
I will investigate, why the Content2 Section has no Identifier itself

**Comment by ls-espirit** [comment]
The missing identifier will be fixed by e-Spirit Core. 
I will mark this issue as resolved as soon as it's released with a new FS Version

**Comment by deangite** [comment]
Close as completed after adding the fix #17
